pytorch/pytorch-native/src/main/native/ai_djl_pytorch_jni_PyTorchLibrary_inference.cc

```diff
@@ -46,6 +46,71 @@ JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_moduleLoad(
   API_END_RETURN()
 }
 
+JNIEXPORT jlong JNICALL Java_ai_djl_pytorch_jni_PyTorchLibrary_moduleLoad__Ljava_io_InputStream_2_3I_3B(
+   JNIEnv* env, jobject jthis, jobject jis, jintArray jarray, jbyteArray arr) {
+    API_BEGIN()
+    jclass is_class = env->GetObjectClass(jis);
+    if (is_class == nullptr) {
+        env->ThrowNew(NULL_PTR_EXCEPTION_CLASS, "Java inputStream class is not found");
+        return -1;
+    }
+    jmethodID method_id = env->GetMethodID(is_class, "read", "([BII)I");
+    if (method_id == nullptr) {
+        env->ThrowNew(ENGINE_EXCEPTION_CLASS, "The read method in InputStream is not found");
+        return -1;
+    }
+    std::ostringstream os;
+    int len = env->GetArrayLength(arr);
+    char* temp = new char[len];
+    int available = 0;
+    while(available != -1) {
+        available = env->CallIntMethod(jis, method_id, arr, 0, len);
+        if (available != -1) {
+            env->GetByteArrayRegion(arr, 0, available, reinterpret_cast<jbyte*>(temp));
```

Comment:
you can remove the temp char array and directly get data from arr

Reply:
No, the os.write requires char* inputs. There is no holder to take that piece of memory. I tried to create the pointer directly without allocating memory and it is just causing Seg fault.

Reply:
ok before we can come up with better solution, let's leave it as it is. add a delete[] arr after you done using arr

Reply:
arr is passed from the Java, you can only delete the local reference